screens/UserScreen.tsx

```diff
@@ -119,7 +124,13 @@ class UserScreen extends React.Component<IProps, IState> {
                     <Text style={styles.orText}>{i18n.or.toUpperCase()}</Text>
                     <View style={styles.buttonWrapper}><StyledButton onPress={() => {this.internalComponentWillUnmount(false);}} inverted={true}>{i18n.goToSignIn.toUpperCase()}</StyledButton></View>
                 </View>
+    </KeyboardAwareScrollView>
             </> : <>
+            <KeyboardAwareScrollView 
```

Comment:
Neither 'sign in' is necessary to be scrollable nor the buttons.

Reply:
The main use in this is for the keyboard aware. I'm not sure how this was working in iOS, but in android the inputs overlapped on opening the keyboard for typing. As in the lower one covered half of the upper one. This was happening here too. Hence, the keyboard aware thing.

Reply:
I see.

Reply:
Sorry, I almost forgot! I agree with you but the buttons are not required to be scrollable, are they?

Reply:
I could change this to be like the inputs are scrollable and the buttons remain fixed, if you like. But then the error messages may get covered by the buttons until the user specifically scrolls down to see them.

Reply:
That is not good. Then leave it as it is now!